src/daemon.rs

```diff
@@ -210,9 +210,7 @@ impl Checker for Daemon {
                 && self.last_below_cpu_usage_percent.is_none()
             {
                 self.last_below_cpu_usage_percent = Some(SystemTime::now());
-            }
-
-            if self.usage <= self.config.high_cpu_threshold.into() {
+            } else {
                 self.last_below_cpu_usage_percent = None;
             }
 
```

Comment:
This doesn't replicate the same effect because now the second if statement (the one replaced by the else) is checking for `self.last_below_cpu_usage_percent.is_none()` which it wasn't before.

Reply:
I see now, that makes sense